service/src/test/java/bio/terra/cbas/util/methods/TestGithubUrlDetailsManager.java

```diff
@@ -0,0 +1,40 @@
+package bio.terra.cbas.util.methods;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.net.URISyntaxException;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.test.context.ContextConfiguration;
+
+@WebMvcTest
```

Comment:
I don't think this is an MVC test. It looks like standard unit tests?

Reply:
Good 'ol copy-and-paste!